fixtures/README.md

```diff
@@ -0,0 +1,40 @@
+# Fixtures Files
+
+## Purpose
+
+We have collected various JS files from a variety of public open source projects on GitHub.
+
+These files help us ensure that future updates to this ESLint config/plugin do not break existing code that passes linting. When a new rule is created, or when an existing rule is updated, these files will help make us aware of scenarios where the updates do not work as expected.
+
+## What Happens If a Plugin/Config Update Breaks Fixtures?
+
+- If the rule is catching something that is not actually a problem:
+  - Reconfigure, disable, or modify the lint rule
+- If the rule is catching something that is actually a problem in the fixtures code:
+  - Whichever is easiest:
+    - [Add `// eslint-disable-next-line some-rule` comments](https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments)
+    - Fix the error manually
+    - Delete the code with the lint error. The fixture files don't have to _work_, so sometimes this is the easiest option.
```

Comment:
Just to clarify here, this would mean to completely remove the entire file from `fixtures`, correct? We don't want to hand-edit these files.

Reply:
Either way is fine. If there is an individual part of the file that is causing trouble, you can just remove it, but if the whole file has a lot of issues, then maybe it is not the best option and the whole file should be removed